packages/block-library/src/post-content/index.php

```diff
@@ -11,6 +11,11 @@
  * @return string Returns the filtered post content of the current post.
  */
 function render_block_core_post_content() {
+	// TODO: Without this temporary fix, an infinite loop can occur.
```

Comment:
Let's add inline comments as to how this fixes the infinite loop and why it occurs.

Reply:
@felixarntz 

>Put in a (temporary) fix for a possible infinite loop. This is not a proper solution, and we might need to find a better way to fix it. I haven't been able to dig to its roots, but basically the block parser continues to iterate the wp_template endlessly when processing the content in a REST API request (which also happens during admin initialization because of preloading).

Interesting, are you sure you didn't have nested post content blocks?

What are the repercussions of this fix. Will returning an empty string like this affect anything? If not, I guess we can keep it.